packages/integration-tests/src/fixtures/infra-envs/index.ts

```diff
@@ -1,14 +1,12 @@
-/* eslint-disable @typescript-eslint/camelcase */
+/* eslint-disable @typescript-eslint/naming-convention */
 
 import { fakeClusterId, fakeClusterInfraEnvId } from '../cluster/base-cluster';
 
 const infraEnv = {
   id: fakeClusterInfraEnvId,
   cluster_id: fakeClusterId,
   href: `/api/assisted-install/v2/infra-envs/${fakeClusterInfraEnvId}`,
-  download_url: `${Cypress.env(
-    'API_BASE_URL',
-  )}/images/${fakeClusterInfraEnvId}?arch=x86_64&type=full-iso&version=4.9`,
+  download_url: `${Cypress.env('API_BASE_URL')}/images/${fakeClusterInfraEnvId}?arch=x86_64&type=full-iso&version=4.9`,
```

Comment:
```suggestion
  download_url: `${Cypress.env('API_BASE_URL')}/images/${fakeClusterInfraEnvId}?arch=x86_64&type=full-iso&version=4.9`,
```

Wouldn't be better to have a function that generates these URLs?

Reply:
It's just a constant, but it could be reused in a few places so I did that